docs/lab/results/result-formats.mdx

```diff
@@ -101,6 +103,120 @@ Interpretation is a string value that can be one of the following:
 3. `Interpretation.CRITICAL` - The result is outside of critical parameters.
     In this case, refer to the [critical values](/lab/workflow/lab-test-lifecycle#critical-results) section.
 
+#### LOINC
+
+As it is possible to test the same biomarkers across different labs, there must be a way to match results across labs. For this, we follow the LOINC standard.
+
+In the `BiomarkerResult` object, you can see two fields `loinc_slug` and `loinc`. These fields refer to the LOINC standard, and are to be used for Customers to be able to match results across labs.
```

Comment:
In the `BiomarkerResult` object, you can see two fields `loinc_slug` and `loinc`. These fields refer to the LOINC standard. Customers should use this standard, so it's possible to match results across different laboratories. 

and are to be used for Customers to be able to match results across labs.